src/con.c

```diff
@@ -419,6 +419,63 @@ bool con_is_hidden(Con *con) {
     return false;
 }
 
+/*
+ * Returns true if the container is maximized in the given orientation.
+ *
+ * If the container is floating or fullscreen, it is not considered maximized.
+ * Otherwise, it is maximized if it doesn't share space with any other
+ * container in the given orientation. For example, if a workspace contains
+ * a single splitv container with three children, none of them are considered
+ * vertically maximized, but they are all considered horizontally maximized.
+ *
+ * Passing "maximized" hints to the application can help it make the right
+ * choices about how to draw its borders. See discussion in
+ * https://github.com/i3/i3/pull/2380.
+ */
+bool con_is_maximized(Con *con, orientation_t orientation) {
+    /* Fullscreen containers are not considered maximized. */
+    if (con->fullscreen_mode != CF_NONE)
+        return false;
```

Comment:
Here and everywhere, please use curly brackets

```suggestion
    if (con->fullscreen_mode != CF_NONE) {
        return false;
    }
```

Reply:
Done (I think I got them all)